api/restHandler/app/AutoCompleteRestHandler.go

```diff
@@ -91,17 +94,17 @@ func (handler PipelineConfigRestHandlerImpl) GetAppListForAutocomplete(w http.Re
 
 	var appIdToObjectMap map[int]string
 	if len(teamId) == 0 {
-		appIdToObjectMap = handler.enforcerUtil.GetRbacObjectsForAllAppsWithMatchingAppName(appName)
+		appIdToObjectMap = handler.enforcerUtil.GetRbacObjectsForAllAppsWithMatchingAppName(appName, helper.AppType(appType))
```

Comment:
Can be solved in v2 : If appType is not one of those we defined, helper.AppType will still pass it. We should handle this in the method it is passed to.